### PR TITLE
Add "Get started with Hadoop Spark" tutorial

### PR DESCRIPTION
**Pull request by WillMoggridge**
Add "Get started with Hadoop Spark" tutorial

## QA
Demo can be found at:
http://tutorials.ubuntu.com-get-started-hadoop.demo.haus/tutorial/get-started-hadoop-spark

### CODE REVIEW

File: tutorials/get-started-hadoop-spark/get-started-hadoop-spark.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+Deployment can take up to 30-45 minutes, as Juju creates new instances in the cloud and sets up the Hadoop Spark cluster components. Pending units are outlined in *orange*. Up and running are outlined in *black*.
+![Status GUI](./images/status-gui.png)
+
+Congratulations! You now have a Hadoop Spark cluster up and running.
+
+
+
+
+## Getting Started
```
Comment (caldav) [inline tutorials/get-started-hadoop-spark/get-started-hadoop-spark.md]: I'd recommend not using "Getting started" as a title for a step that is not the first step of a tutorial. What about something more descriptive, like "Installing Juju"?

Comment (elvirafigueras) [inline tutorials/get-started-hadoop-spark/get-started-hadoop-spark.md]: +1 on the Getting Started thought. Changes made. Thanks!

**Comment by WillMoggridge** [comment]
@elvirafigueras I updated the note to the format of the positive info box, following the style that we have used in other tutorials. For example: https://tutorials.ubuntu.com/tutorial/advanced-snap-usage#1

File: tutorials/get-started-hadoop-spark/get-started-hadoop-spark.md (hunk 1 above)
Comment (elvirafigueras) [inline tutorials/get-started-hadoop-spark/get-started-hadoop-spark.md]: @caldav true - we'll use the original image for now and see to update it in the next iteration

**Comment by elvirafigueras** [comment]
@WillMoggridge that's cool thanks!

**Comment by WillMoggridge** [comment]
@caldav It seems the team would like to proceed with the image as it is and get the tutorial released.

I would definitely prefer to get your OK on the review before moving foward. Would you be OK with the image as it is? Or if not, could we release this and move the discussion into an issue?

**Comment by caldav** [comment]
It's definitely more a nitpick than a blocker, +1 from me

**Pull request merged**